base/uk.ac.stfc.isis.ibex.e4.ui.perspectiveswitcher/src/uk/ac/stfc/isis/ibex/e4/ui/perspectiveswitcher/controls/ScriptServerButtonViewModel.java

```diff
@@ -0,0 +1,75 @@
+package uk.ac.stfc.isis.ibex.e4.ui.perspectiveswitcher.controls;
+
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+
+import org.eclipse.swt.SWT;
```

Comment:
Are these really needed? They're not being used and it runs fine without them.

Reply:
Removed.